### ISSUE TITLE
PyOpenSSL: Fails to import after installation with the provided instructions

### ISSUE THREAD

**Issue by davidfstr** [shazow/urllib3#839]
The [Security documentation page] indicates that to use PyOpenSSL that you must first install:

* pip3 install pyopenssl ndg-httpsclient pyasn1

And then in a Python interpreter run:

* import urllib3.contrib.pyopenssl
* urllib3.contrib.pyopenssl.inject_into_urllib3()

However on Python 3.4.4 (the latest 3.4.x) on OS X 10.11 (the latest OS X) I get the following traceback when executing `import urllib3.contrib.pyopenssl`:

```
Traceback (most recent call last):
  File "/Users/davidf/Projects/webcrystal/venv/lib/python3.4/site-packages/urllib3/contrib/pyopenssl.py", line 60, in <module>
    from socket import _fileobject
ImportError: cannot import name '_fileobject'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/davidf/Projects/webcrystal/venv/lib/python3.4/site-packages/urllib3/contrib/pyopenssl.py", line 63, in <module>
    from urllib3.packages.backports.makefile import backport_makefile
ImportError: No module named 'urllib3.packages.backports'
```

Perhaps there is some missing step in the documentation?

[Security documentation page]: https://urllib3.readthedocs.org/en/latest/security.html#openssl-pyopenssl

**Comment by Lukasa** [comment]
Nope, that looks like a real bug to me!

In particular, this looks like we incorrectly packaged the release. We added a new module that appears to not be being collected for installation.

We'll be able to get a new release out for this specific problem quite quickly, I suspect. @shazow if I push up a fix, are you interested in running out a really quick .1 release with the fix?

**Comment by Lukasa** [comment]
Fix available in #841.

**Comment by davidfstr** [comment]
Verified as fixed.
